status/status_ext_test.go

```diff
@@ -203,3 +206,57 @@ func (s) TestStatusDetails(t *testing.T) {
 		})
 	}
 }
+
+// TestStatus_ErrorDetailsMessageV1 verifies backward compatibility of the
+// status.Details() method when using protobuf code generated with only the
+// MessageV1 API implementation.
+func (s) TestStatus_ErrorDetailsMessageV1(t *testing.T) {
+	details := []protoadapt.MessageV1{
+		&tpb.SimpleMessage{Data: "abc"},
+		&testpb.Empty{},
```

Comment:
According to the test comments, should `testpb.Empty` be removed ?

Reply:
Yes, removed. I wanted to keep a test case for ensuring v1 + v2 messages are also handled correctly, added a new test function for that later.